user_manual/groupware/calendar.rst

```diff
@@ -235,6 +235,19 @@ The ``Empty trash bin`` buttons will wipe all trash bin contents in one step.
 
 .. _calendar-working-hours:
 
+Responding to invitations
+-------------------------
+
+As of Calendar v3.1.0 you can respond to invitations inside the app. Click on the event and select your participation status. You can respond to an invitation by accepting, declining or accepting tentatively.
```

Comment:
I would just make the assumption that 25/24 installations use the latest compatible Calendar app